<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:rgb(240, 235, 190)
}
QPushButton{
	background-color:rgb(182, 217, 157);
	border-style: outset;
	border-radius: 10px;
	border-color: beige;
	font: bold 14px;
	padding: 6px;		
}
QPushButton:pressed{
	background-color:rgb(147, 204, 105);
	border-style: outset;
	border-radius: 10px;
	border-color: beige;
	font: bold 14px;
	padding: 6px	
}
QSlider::handle:horizontal {
	background: rgb(182, 217, 157);
	padding: 6px;
	border-style: outset;
	border-radius: 10px;
	border-color: beige;
	font: bold 14px;
	margin: 0 -4px;
	
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Такой пользователь уже имеется!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_but">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
